packages/components/nodes/GoogleDocs/GoogleDocs.ts

```diff
@@ -0,0 +1,295 @@
+    description?: string;
+    version: number;
+    icon: string;
+    incoming: number;
+    outgoing: number;
+    actions?: INodeParams[];
+    credentials?: INodeParams[];
+    inputParameters?: INodeParams[];
+
+    constructor() {
+        this.label = 'GoogleDocs';
+        this.name = 'googleDocs';
+        this.icon = 'gdocs.svg';
+        this.type = 'action';
+        this.version = 1.0;
+        this.description = 'Execute GoogleDocs operations';
+        this.incoming = 1;
+        this.outgoing = 1;
+
+        this.actions = [
+            {
+                label: 'Operation',
+                name: 'operation',
+                type: 'options',
+                options: [
+                    {
+                        label: 'Create New Document',
+                        name: 'create',
+                        description: 'Create a new document'
+                    },
+                    {
+                        label: 'Get All Values',
+                        name: 'getAll',
+                        description: 'Get all values from a document'
+                    },
+                    {
+                        label: 'Update a Document',
+                        name: 'update',
+                        description: 'Update a document'
+                    }
+                ],
+                default: 'create'
+            }
+        ] as INodeParams[];
+
+        this.credentials = [
+            {
+                label: 'Credential Method',
+                name: 'credentialMethod',
+                type: 'options',
+                options: [
+                    {
+                        label: 'Google Docs OAuth2',
+                        name: 'googleDocsOAuth2Api'
+                    }
+                ],
+                default: 'googleDocsOAuth2Api'
+            }
+        ] as INodeParams[];
+
+        this.inputParameters = [
+            {
+                label: 'Document Name',
+                name: 'documentName',
+                type: 'string',
+                optional: true,
+                description: 'Name of the document to create. Default to Untitled document.',
+                show: {
+                    'actions.operation': ['create']
+                }
+            },
+            {
+                label: 'Document',
+                name: 'documentId',
+                type: 'asyncOptions',
+                loadMethod: 'getAllDocsFromDrive',
+                hide: {
+                    'actions.operation': ['create']
+                }
+            },
+
+            /**
+             *  batch update
+             */
+
+            {
+                label: 'Requests',
+                name: 'requests',
+                description:
+                    "update a document. You can simply add one reequest data or add multiple request data. If request format is invalid, document won't be updated.",
+                type: 'json',
+                placeholder: `[
```

Comment:
I tried the placeholder it doesn't work. Perhaps its better to put something that was working so user can try it out on the first try. This was what worked for me on a newly created blank doc: 
```
[
    {
      "insertText": {
        "location": {
          "index": 1
        },
        "text": "hihi"
        }
    }
]
```

Reply:
I fixed the placeholder.
I checked the placeholder is executable json.
Here is how it looks.
<img width="1430" alt="Screen Shot 2022-10-15 at 11 13 30 AM" src="https://user-images.githubusercontent.com/69574727/196001957-0d346291-f798-4f97-9045-d62df32d45dc.png">